swift_scality_backend/diskfile.py

```diff
@@ -481,6 +482,16 @@ def open(self):
         if metadata is None:
             raise swift.common.exceptions.DiskFileDeleted()
         self._metadata = metadata or {}
+
+        try:
+            x_delete_at = int(self._metadata['X-Delete-At'])
+        except KeyError:
+            pass
+        else:
```

Comment:
But we don't have the concept of "quarantine". If X-Delete-At is not an int it means that the Ring corrupted the data, or suffered from bit rote. What can we do about it ?